### ISSUE TITLE
how to get newly created chain info when calling createReplyChain?

### ISSUE THREAD

**Issue by richard1122** [OfficeDev/BotBuilder-MicrosoftTeams-node#12]
Hi, I'm using `createReplyChain` to send messages to a new chain (in the channel).
```typescript
      const activity = {
        type: ActivityTypes.Message,
        attachments: [{
          contentType: 'application/vnd.microsoft.card.adaptive',
          content: buildFeedbackDialog(message),
        }],
      } as Activity
      const resource = (await adapter.createReplyChain(context, [activity]))[0]
      console.log(`Add record: ${resource.id}`)
```

After that, the only thing I got is `resourceId`

![Screenshot from 2019-05-09 17-55-25](https://user-images.githubusercontent.com/2534277/57444992-f8163d00-7283-11e9-89e4-5970bba77152.jpg)

So after I reply to the message created by the bot and received onMessage webhook, how could I know which 'initial activity' or 'reply chain' the replied message activity belongs to?

Or knowing the relation between the bot's initial activity and the user reply message?

Thanks.

**Comment by robin-liao** [comment]
Hi @richard1122 ,

You may check out the conversation object (and its id) from the incoming activity, by `context.activity.conversation.id`, and the conversation id looks like:

`19:fa6e105024cd4575ab5b6b81d6ebcda2@thread.skype;messageid=1557438940572`

which means __Channel id + Reply chain root message id__:
`19:fa6e105024cd4575ab5b6b81d6ebcda2@thread.skype` is channel id
`messageid=1557438940572` is the parent / root message id of the current reply chain.

**Comment by richard1122** [comment]
Hi @robin-liao , 

Thanks for your reply.

I know that I could extract message_id from when receiving updates from the context.

But when creating the reply chain ( bot send the first message ), no message_id is returned.

The only response from `createReplyChain` is:

```typescript
export interface ResourceResponse {
    /**
     * Id of the resource
     */
    id: string;
}
```

The id is `1:1xel-XL7Sppbet .....`

Which means I'm not sure how to get the message_id when creating reply chain.

Thanks.

**Comment by robin-liao** [comment]
@richard1122  you're right!! After some investigations, I found it's indeed a design flaw. It should've return new message id. Reopen this issue and will fix bug soon. Thanks!
